frontend/tool/webpack.prod.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = merge(baseWebpackConfig, {
     plugins: [
         new webpack.DefinePlugin({
             'process.env': {
-                'NODE_ENV': 'production'
+                NODE_ENV: '"production"'
```

Comment:
Oh I hate this guy. Every version of webpack requires this to be done differently. Glad you found the right way

Reply:
so the one ```'NODE_ENV': '"dev"'``` in webpack.dev.config.js is it ok?

Reply:
Yes, that should be fine. We can always develop with Vue without trouble. It is the prod environment having issues.